examples/framework.rs

```diff
@@ -120,7 +76,7 @@ async fn run_async<E: Example>(title: &str) {
 
     let mut sc_desc = wgpu::SwapChainDescriptor {
         usage: wgpu::TextureUsage::OUTPUT_ATTACHMENT,
-        format: wgpu::TextureFormat::Bgra8UnormSrgb,
+        format: wgpu::TextureFormat::Bgra8Unorm,
```

Comment:
Agreed, I mentioned this a bit in https://github.com/gfx-rs/wgpu-rs/pull/193#discussion_r392665836 – I wasn't sure what the plans are for supported swapchain formats and whether srgb would be supported initially on the web (e.g. https://github.com/WICG/canvas-color-space).

I'm not sure if there's a way to query whether srgb is available for the swapchain. We do have `getSwapChainPreferredFormat` which could give a weak signal of whether srgb is supported for the preferred format. I guess the only fall back path is to use `cfg` for now, although I was trying to avoid `cfg` in examples.

Reply:
I used `cfg` for now